cfssl/README.md

```diff
@@ -0,0 +1,53 @@
+# Agent Check: cfssl
+
+## Overview
+
+This check monitors [cfssl][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+The cfssl check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `cfssl.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your cfssl performance data. See the [sample cfssl.d/conf.yaml][4] for all available configuration options.
+
+2. [Restart the Agent][5].
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `cfssl` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+The cfssl integration does not include any metrics.
+
+### Events
+
+The cfssl integration does not include any events.
+
+### Service Checks
+
+See [service_checks.json][8] for a list of service checks provided by this integration.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][9].
```

Comment:
```suggestion
Need help? Contact [Datadog support][9].
```

Reply:
Could you confirm if there is a link for customers to reach out to cfssl support in addition to Datadog support?

Reply:
cfssl is open-source software provided 'as is' by Cloudflare − I don’t know of any kind of support provided by Cloudflare about it.